department-of-veterans-affairs/va.gov-cms
# VACMS-4541: Limit VC selector to admin option.

## Description
closes #4541

## Testing done
Visual

## QA steps
 - As an administrator, visually verify that selection chosen in `Section` field limits the Vet Center choice in `Vet Center` select field.:
    - [x]  go to https://pr7952-js5xh7o5ugdclq6isrbyxsjjowzkmlaf.ci.cms.va.gov/node/add/vet_center_locations_list, 
    - [x] https://pr7952-js5xh7o5ugdclq6isrbyxsjjowzkmlaf.ci.cms.va.gov/node/add/vet_center_outstation 
    - [x] https://pr7952-js5xh7o5ugdclq6isrbyxsjjowzkmlaf.ci.cms.va.gov/node/add/vet_center_mobile_vet_center
